### DISCUSSION TITLE
4.6.1版本视频播放时不能右键查看视频播放码率信息了，是bug吗

### DISCUSSION THREAD

**Discussion by Farconquer** [ywmoyue/biliuwp-lite#127] [General]

**Comment by ywmoyue** [comment]
不是bug，变更手势实现后暂时将视频信息放到播放设置中了
![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/7735f434-bee3-4350-9789-d5bbc2145b1a)
